microsoft-edge/dualengine/concepts/adapter-dll.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Understanding the DualEngine Adapter DLL
+description: Understanding the requirements and uses of DualEngine Adapter DLL.
+author: MSEdgeTeam
+ms.author: edgededev
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.localizationpriority: high
+ms.technology: windows-integration
+ms.date: 11/06/2023
+---
+
+# Understanding the DualEngine Adapter DLL
+The DualEngine API is an API that can only be used from inside the Internet Explorer process.
+As such in order to make use of the API it is necessary to create a plugin DLL that Internet Explorer will load.
+This DLL will host your code that uses the DualEngine API as well as presumably any code you need to communicate

```

Comment:
```suggestion
This DLL will host your code that uses the DualEngine API, as well as any code you need to communicate
```

Reply:
This DLL hosts your code that uses the DualEngine API, as well as any code that's needed for communicating with your main application;